.pre-commit-config.yaml

```diff
@@ -23,3 +23,17 @@ repos:
     rev: v3.4.0
     hooks:
       - id: pyupgrade
+  - repo: https://github.com/dhruvmanila/remove-print-statements
```

Comment:
Pay attention, this doesn't seem super-maintained (little activity, unknown adoption...)

It might be fine, but we should drop it as soon as it will cause issues

Reply:
If it will cause issues I'll maybe try to link the pre-commit to the test_print.py... I think it's better if that check is done before committing

Reply:
I agree that it is a better solution, also because the other one consisted I'm copy-pasting the same test across different repos.

In case something goes wrong, we can always fork that repo and fix it by ourselves (that is still easier than making the hook ourselves).

I just wanted to warn you about this option, since it might not work forever out of the box. But I agree that in the current situation it's a suitable solution.